ng-dev/pr/merge/strategies/api-merge.ts

```diff
@@ -110,24 +153,24 @@ export class GithubApiMergeStrategy extends AutosquashMergeStrategy {
       );
     }
 
+    // Refresh the target branch the PR has been merged into through the API. We need
+    // to re-fetch as otherwise we cannot cherry-pick the new commits into the remaining
+    // target branches. Also, this is needed fo the merge comment to get the correct commit SHA.
+    this.fetchTargetBranches([githubTargetBranch]);
+
     // If the PR does not need to be merged into any other target branches,
     // we exit here as we already completed the merge.
     if (!cherryPickTargetBranches.length) {
+      await this.createMergeComment(pullRequest, targetBranches);
+
       return;
     }
 
-    // Refresh the target branch the PR has been merged into through the API. We need
-    // to re-fetch as otherwise we cannot cherry-pick the new commits into the remaining
-    // target branches.
-    this.fetchTargetBranches([githubTargetBranch]);
-
-    // Number of commits that have landed in the target branch. This could vary from
-    // the count of commits in the PR due to squashing.
-    const targetCommitsCount = method === 'squash' ? 1 : pullRequest.commitCount;
```

Comment:
Is this no longer needed?

Reply:
Nope as now I am using `pullRequestCommitCount`